node/src/components/consensus/config.rs

```diff
@@ -24,6 +24,10 @@ pub struct Config {
     pub unit_hashes_folder: PathBuf,
     /// The duration for which incoming vertices with missing dependencies are kept in a queue.
     pub pending_vertex_timeout: TimeDiff,
+    /// If the current era's protocol state has not progressed for this long, shut down.
+    pub standstill_timeout: TimeDiff,
+    /// Log inactive or faulty validators periodically, with this interval.
+    pub log_participation_interval: TimeDiff,
```

Comment:
Hopefully, my PR merges before this so you should update this and move to highway-specific config.

Reply:
I know, I already thought about it. But I selflessly, heroically, approved it anyway so you could merge first. :innocent: